pkg/kube/client_test.go

```diff
@@ -148,10 +150,19 @@ func TestUpdate(t *testing.T) {
 	reaper := &fakeReaper{}
 	rf := &fakeReaperFactory{Factory: f, reaper: reaper}
 	c := &Client{Factory: rf}
-	if err := c.Update(api.NamespaceDefault, objBody(codec, &listA), objBody(codec, &listB), false); err != nil {
+	if err := c.Update(api.NamespaceDefault, objBody(codec, &listA), objBody(codec, &listB), false, 0, false); err != nil {
 		t.Fatal(err)
 	}
-
+	// TODO: Find a way to test methods that use Client Set
```

Comment:
The latter. @technosophos and @adamreese asked me to keep it in because this should be tested but can't be